std/assembly/util/string.ts

```diff
@@ -1,6 +1,419 @@
 import { itoa, dtoa, itoa_stream, dtoa_stream, MAX_DOUBLE_LENGTH } from "./number";
 import { ipow32 } from "../math";
 
+// All tables stored as two staged lookup tables (aka static tries) due to unicode has
+// a wide range of symbols [0 - 0x110000] which can't represent in memory as is.
+// Multistage Table approach described in Unicode spec ch 5, p.196:
+// https://www.unicode.org/versions/Unicode12.0.0/ch05.pdf
+// All tables generated by forked tools created by Rich Felker for musl:
+// https://github.com/MaxGraey/musl-chartable-tools/tree/case-ignorable
```

Comment:
Perhaps
```
// All tables are stored as two staged lookup tables (static tries)
// because the full range of Unicode symbols can't be efficiently
// represented as-is in memory (see Unicode spec ch 5, p.196):
// https://www.unicode.org/versions/Unicode12.0.0/ch05.pdf
// Tables have been generated using these forked musl tools:
// https://github.com/MaxGraey/musl-chartable-tools/tree/case-ignorable

// Lookup table to check if a character is alphanumeric or not
```

Reply:
Done!